docusaurus.config.js

```diff
@@ -63,6 +63,10 @@ const config = {
   themeConfig:
     /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
     ({
+      colorMode: {
+        defaultMode: 'light',
+        disableSwitch: true,
+      },
```

Comment:
I don't think we should disable color mode switch.

Reply:
The UI designer did not provide dark mode.
I'm also not good at matching colors, so the color mode switch is disabled.
BTW, there is no dark mode on the home page of some other apache projects. I think it might not be a necessary requirement.

Reply:
Well. It's not required to maintain color mode toggle and it's OK to keep light mode only.